lib/util/resolve-file.ts

```diff
@@ -0,0 +1,15 @@
+import findUp from 'find-up';
+import { join } from 'upath';
+
+/**
+ * Resolve path for a file relative to renovate root directory (our package.json)
+ * @param file a file to resolve
+ */
+export async function resolveFile(file: string): Promise<string> {
```

Comment:
Does this belong in `lib/util/fs`?

Reply:
Good point, just extracted it to fix circular dependency